nominal/core/dataset.py

```diff
@@ -140,6 +140,12 @@ def add_to_dataset_from_io(
         )
         self._clients.ingest.trigger_file_ingest(self._clients.auth_header, request)
 
+    def get_channel(self, name: str) -> Channel:
```

Comment:
added helper function for getting a single channel by name

Reply:
kinda want to rename the other function to `search_channels` since it is a search function, not "get by name" or "get by rid" etc

Reply:
since this is "name" maybe this should be `get_channel_by_name()`?

Reply:
`get_channel_by_rid` doesn't really make sense on a single dataset, but it does on the client. so we could have:

```python
client.get_channel()  # by rid
dataset.get_channel()  # by name
```

but might be kinda whack to have these with different semantics?

Reply:
the other `get_blah()` methods take a `rid`, so... maybe 

```python
client.get_channel()  # by rid
dataset.get_channel_by_name()  # by name
```

is the best

Reply:
alternatively we could force a keyword arg e.g. `get_channel(name="name")` and `get_channel(rid="rid")`

Reply:
Probably fine to just keep this function scoped to a dataset for now